### DISCUSSION TITLE
Chest always contains armor

### DISCUSSION THREAD

**Discussion by Canarious** [jordanbtucker/totk-no-loot-boxes#14] [Q&A] [answered]
**Urbosa** started out dropping the **Gerudo-Champion Fabric**, then the **Vah Naboris Divine Helm**, **Gerudo Scimitar** and **Gerudo Shield** like it was supposed to. After save and load of the game, it started dropping the, **Vah Naboris Divine Helm** only.
Is it possible to have her drop the **Gerudo Scimitar**, **Gerudo Shield**, **Gerudo Claymore**, **Gerudo Bow**, and **Gerudo Spear** as well if not then half-n-half with **Smash Brothers Ganondorf**?

**Comment by jordanbtucker** [comment]
I just tested this and I could not reproduce the issue. However, I'm on TotK v1.1.1 and I haven't tested v1.1.0 yet.

- Which version of TotK are you on?
- Which version of No Loot Boxes are you using?
- Are you playing on Switch, Yuzu, or Ryujinx?

**Reply by Canarious** [reply]
Thank you kindly for your reply.
I'm currently on Ryujinx using No Loot Boxes 2.0 version 1.1.1

**Reply by jordanbtucker** [reply]
Thanks for the info. I'm still having trouble tracking down this bug. You're not the only one who has reported this. Would you be willing to post your save files? The `user` folder at `%APPDATA%\Ryujinx\bis\user` should be enough.

**Reply by Canarious** [reply]
[user.zip](https://github.com/jordanbtucker/totk-no-loot-boxes/files/11523617/user.zip)

**Reply by RuafuMan-Obs** [reply]
Got the same problem on drop table ,only drop clothes on every amiibo contains.
Those doesnt contain clothes work good ,like Ganondorf and TP Zelda,only drop Twilight Claymore / Twilight Bow

**Reply by jordanbtucker** [reply]
Thanks @Canarious. I'm able to reproduce the issue now. Perhaps there's a point in the game at which it starts giving you amiibo armor even if you already have it. All of my tests have been shortly after completing the Great Sky Island. I'll have to do some testing.

In the meantime, if there are any rare weapons you want to obtain, like the White Sword of the Sky from the Skyward Sword Link amiibo, you can use [v1.2.0](https://github.com/jordanbtucker/totk-no-loot-boxes/releases/tag/v1.2.0) of this mod, which assigns all rare items with equal probability.

If I can't solve this, then I'll publish a v2.0.1 which will assign equal probability to all armor and weapons. At least the fabric can still be obtained without duplicates.

**Reply by jordanbtucker** [reply]
@Canarious Quick question. Did you obtain any of the helms without using amiibo?

**Reply by Canarious** [reply]
My apologies for the late reply @jordanbtucker I was at work. I see the update you released and I appreciate your work very much. I'm not sure if my answer will help you since you've already released your patch but the answer is yes I did obtain the **Vah Medoh** helmet from a chest in the north west mountains.

**Reply by jordanbtucker** [reply]
Thanks! Your answer still helps because I still want to track down what causes the bug.

**Comment by jochip** [comment]
Not sure if this is a known issue but when i activate a Sheik Smash bros amiibo i keep getting Sheik masks, i've pulled four of them in a row and i'm unable to receive the rest of the amiibo drops (Eightfold blade and Mind's eye shield)

![image](https://github.com/jordanbtucker/totk-no-loot-boxes/assets/134120024/dedefb92-00c1-4b17-8ead-ddb7a77d5cf6)

Log: 
[yuzu_log.txt](https://github.com/jordanbtucker/totk-no-loot-boxes/files/11523188/yuzu_log.txt)

**Reply by jordanbtucker** [reply]
I've been seeing reports about this, but I haven't been able to duplicate the issue on Switch, Yuzu, or Ryujinx with any version of TotK. One difference is that I haven't tried Yuzu EA, so maybe I need to test that too.

Thanks for the logs. I noticed you have some other mods installed that are likely FPS mods. These should be compatible, but for troubleshooting sake, can you please make sure all other mods, patch files, and cheats are disabled and try again.

**Reply by jochip** [reply]
Here it is running only the NoLootboxes mod, another two Sheik masks hahah
![image](https://github.com/jordanbtucker/totk-no-loot-boxes/assets/134120024/1ab92163-f182-43ce-8fd2-adb5e9d47164)
[yuzu_log.txt](https://github.com/jordanbtucker/totk-no-loot-boxes/files/11523210/yuzu_log.txt)

**Reply by jordanbtucker** [reply]
Thanks for the log file. Unfortunately, I still haven't been able to track down this bug. Are you willing to post your save files? The `user` folder at `%APPDATA%\yuzu\nand\user` should be enough.

**Reply by jochip** [reply]
Sorry for th late reply, I'm not sure if it was necessary but i took the liberty of recreating the bug and summoning more Sheik drops to bump it up to the most recent event right before saving. 
Save file: 
[save-5-21-2023.zip](https://github.com/jordanbtucker/totk-no-loot-boxes/files/11524732/save-5-21-2023.zip)

I'll be using the v2.0.1 update for now tho, thank you!

Also, not sure if it's relevant but just in case, I am following this thread on Reddit when it comes to troubleshooting and updating:
https://www.reddit.com/r/NewYuzuPiracy/comments/13gh9ts/comment/jkf7rku/?utm_source=share&utm_medium=web2x&context=3

**Comment by jordanbtucker** [comment, marked as answer]
I released [v2.0.1](https://github.com/jordanbtucker/totk-no-loot-boxes/releases/tag/v2.0.1), which should partially fix this issue by making armor have the same drop rate as weapons, bows, and shields. This will not prevent armor from dropping even when you already have it, but it will prevent armor from being the only thing that drops.
